test/extensions/voting-rep.js

```diff
@@ -3159,4 +3161,165 @@ contract("Voting Reputation", (accounts) => {
       expect(motion.altTarget).to.equal(oneTxPayment.address);
     });
   });
+
+  describe("interaction with other extensions", async () => {
+    it("can release a staked expenditure via a motion, even if owner has no permissions", async () => {
```

Comment:
I don't think this test is particularly necessary, the second test makes this one redundant.

Reply:
To a large extent, I do agree. It's still legitimate behaviour though which e.g. the block ingestor will need to support, so I'm somewhat inclined to leave it in as a demonstration of that.